api/externals/repository/sponsor_repository.go

```diff
@@ -48,10 +60,12 @@ func (sr *sponsorRepository) Create(
 	address string,
 	representative string,
 ) error {
-	query := `
-		INSERT  INTO
-			sponsors (name, tel, email, address, representative)
-		VALUES ("` + name + `","` + tel + `","` + email + `","` + address + `","` + representative + `")`
+	dialect := goqu.Dialect("mysql")
+	ds := dialect.Insert("sponsors").Rows(goqu.Record{"name": name, "tel": tel, "email": email, "address": address, "representative": representative})
```

Comment:
```suggestion
	ds := dialect.Insert("sponsors").
		Rows(goqu.Record{
			"name":           name,
			"tel":            tel,
			"email":          email,
			"address":        address,
			"representative": representative})
```
長くなりそうならこんな感じにしてもよさそう
lintされないかな勝手に

Reply:
長くなりそうだなー、
lintあるならこっちでもいいけど、整形するのにコストかかるな
既存でもそこまで読みにくくないけどどう？

Reply:
https://github.com/segmentio/golines
長くなるの折り返しならこれ良さそう

Reply:
![image](https://github.com/user-attachments/assets/65297f04-3035-49b2-aa5c-faf524ff4ea7)
innerjoinが若干見切れるのが悲しい気持ちになるけど、だいぶ改善されてよさそう